src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/WebTestBase.kt

```diff
@@ -14,7 +14,7 @@ abstract class WebTestBase : IntegrationTestBase() {
   @Autowired
   internal lateinit var jwtAuthorisationHelper: JwtAuthorisationHelper
 
-  internal fun WebTestClient.RequestHeadersSpec<*>.authorised(roles: List<String> = listOf(Roles.ROLE_QUEUE_ADMIN)): WebTestClient.RequestBodySpec {
+  internal fun WebTestClient.RequestHeadersSpec<*>.authorised(roles: List<String> = listOf(Roles.QUEUE_ADMIN)): WebTestClient.RequestBodySpec {
```

Comment:
Not relevant to this PR, and not a big deal, but I wonder if we should have this role as a default? It feels like something we did while migrating tests to use `WebTestClient` just to keep them green rather than a conscious decision. It's not as if any requests to our service apart from those originating from the DLQ recycling cronjob will have this role